README.rst

```diff
@@ -36,10 +36,7 @@ Installation
 
 From a command shell run ::
 
-  pip install nose2pytest
-
-This puts an executable file in ``<python-root>/Scripts`` with *python-root* being the root folder of the 
-Python installation from which ``pip`` was run.
+  pipx install --python 3.11 nose2pytest@git+https://github.com/pytest-dev/nose2pytest
```

Comment:
The reason I used the install from GitHub was to ensure that the issue I was encountering was not fixed at HEAD. Probably you still want to stick with installing from PyPI and just make sure releases are published there with any updates.

```suggestion
  pipx install --python 3.11 nose2pytest
```

Reply:
As discussed on #26, should we encourage `pipx run` instead of `pipx install` given the transitory nature of using this tool?

Reply:
Updated for `pipx run --python 3.11 nose2pytest path/to/dir/with/python_files`.

We will need to ensure there is a new release to PyPI.